redisinsight/ui/src/components/base/layout/loading-content/loading-content.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { HTMLAttributes } from 'react'
+import styled, { keyframes } from 'styled-components'
+
+export type LineRange = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10
+
+export interface LoadingContentProps extends HTMLAttributes<HTMLDivElement> {
+  lines?: LineRange
+}
+
+const loadingAnimation = keyframes`
+  0% {
+    transform: translateX(-53%);
+  }
+
+  100% {
+    transform: translateX(0);
+  }
+`
+
+export const StyledLoadingContent = styled.span.attrs(() => ({}))<
+  React.HtmlHTMLAttributes<HTMLSpanElement>
+>`
+  display: block;
+  width: 100%;
+`
```

Comment:
Why do you need attrs?
```suggestion
export const StyledLoadingContent = styled.span<React.HtmlHTMLAttributes<HTMLSpanElement>>`
  display: block;
  width: 100%;
`
```